src/main/java/net/dv8tion/jda/api/requests/restaction/interactions/InteractionCallbackAction.java

```diff
@@ -46,31 +48,35 @@ public interface InteractionCallbackAction<T> extends RestAction<T>
     enum ResponseType
```

Comment:
The part about this being only used internally is outdated now

Reply:
So you're saying we should put it in its own file?

Reply:
No, the docs are just outdated